components/style/gecko/media_queries.rs

```diff
@@ -474,47 +474,55 @@ impl Expression {
                          -> Result<Self, ParseError<'i>> {
         input.expect_parenthesis_block()?;
         input.parse_nested_block(|input| {
-            let ident = input.expect_ident()?;
-
-            let mut flags = 0;
-            let result = {
-                let mut feature_name = &*ident;
+            // FIXME: remove extra indented block when lifetimes are non-lexical
```

Comment:
There’s a draft RFC and prototype: http://smallcultfollowing.com/babysteps/blog/2017/07/11/non-lexical-lifetimes-draft-rfc-and-prototype-available/ . I’m hopeful it’ll land in stable in 2018.